examples/lustre/test.bats

```diff
@@ -0,0 +1,37 @@
+load "${CHTEST_DIR}/common.bash"
+
+setup () {
+    scope standard
+    if [[ -z "${ch_lustre}" ]]; then
+        skip 'no lustre to bind mount'
+    fi
+    prerequisites_ok lustre
+
+    # Check lustre directory exists
+    if [[ ! -e "$ch_lustre" ]]; then
+        skip "${ch_lustre} does not exist"
+    fi
+    # Check lustre directory is a directory
+    if [[ ! -d "$ch_lustre" ]]; then
+        skip "${ch_lustre} is not a directory"
+    fi
+}
+
+@test "${ch_tag}/write" {
+    binds=${ch_lustre}:/lustre
```

Comment:
1. Is there a reason to use `/lustre` rather than existing `/mnt`?
2. I recommend to set `binds` once at the top of the file, to avoid repeating it.

Reply:
Just for human brain continuity. So no technical reason. I think I will make a change to address this in a different way.